templates/engage/thank_you_ad.html

```diff
@@ -0,0 +1,24 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}Ubuntu 16.04下载, Ubuntu 18.04下载, Ubuntu服务器下载， Ubuntu系统下载， Ubuntu中文官网， Ubuntu OpenStack, MicroK8s, Ubuntu容器{% endblock %}
+
+{% block title %}将Ubuntu桌面与Active Directory集成{% endblock %}
+
+{% block content %}
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-7">
+<h1 class="p-heading--two">谢谢！</h1>
+<h3>你的白皮书现已可下载...</h3>
+<p><a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Canonical%20-%20Active%20Directory_7.5.19-CN.pdf" class="link-cta-ubuntu cta-large" target="download">立即下载</a></p>
```

Comment:
please fix indentation.

Reply:
Changed to col-8+col-4, is it ok ?

Reply:
@zprood sorry, is not what I meant. Is just the code if you can indent it properly. Like h1, h3 & p to start 2 spaces to the right of the parent div.

Reply:
@solazio aha, got your point. The tag's position changed.